src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -482,54 +491,48 @@ void StartImportingBasicBlock(BasicBlock basicBlock)
             {
                 Check(basicBlock.EntryStack == null || basicBlock.EntryStack.Length == 0, VerifierError.TryNonEmptyStack);
 
-                for (int i = 0; i < _exceptionRegions.Length; i++)
```

Comment:
Single basic block can be start of multiple try blocks, e.g. 

```
try
{
    Console.WriteLine("Hello world");
}
catch (OutOfMemoryException)
{
    Console.WriteLine("OutOfMemoryException");
}
catch (InvalidOperationException)
{
    Console.WriteLine("InvalidOperationException");
}
```

So there needs to be some kind of loop to trigger all filters/handlers unfortunately.